gapipy/models/base.py

```diff
@@ -104,15 +104,15 @@ def _set_price_field(self, field, value):
     def _model_cls(self, field):
         # Find what model class this field is in reference to by plucking it
         # from the set of all fields that allow this type of definition.
-        fields = (self._model_fields
```

Comment:
I'm a big fan of automatically formatting code, but this file hasn't otherwise been touched for implementing the feature, so it's a bit strange to have these formatting changes as part of this PR.

Reply:
> I'm a big fan of automatically formatting code

👍 👍 👍 

gapipy is probably small enough that we can just blacken it in one go once we're ready to enforce that for new changes in CI.

I've got some vague plans to get the gapipy tests running again for PRs either in Circle or maybe GH Actions (cause it is free for public repos), once we're at that point maybe it's easy enough to enforce formatting.

I know that the codebase needs to remain 2/3 compatible for now, though.

--

As far as this particular change in this PR that you've highlighted: yeah, I could take it or leave it.

@marz619 you have strong feelings about keeping this bit in? If not I can revert it.

Reply:
> you have strong feelings about keeping this bit in? If not I can revert it.

nope, revert away.